### DISCUSSION TITLE
delay-0

### DISCUSSION THREAD

**Discussion by jamiebuilds-signal** [tailwindlabs/tailwindcss#10293] [Ideas]
This is an occasional pattern I use:

```html
<div id="titlebar" class="... bg-transparent bg-indigo-600 transition-colors delay-300 active:delay-0" />
```

- Default state is `bg-transparent`
- Immediately changes the color to `bg-indigo-600` upon `active:` 
- Delays the transition back to default state of `bg-transparent` by `delay-300`

This is useful for making something feel instantly responsive, but less visually distracting when you're clicking it multiple times.

I found it surprising that Tailwind did not have this baked in

**Comment by jamiebuilds-signal** [comment]
Apologies, I had searched issues but not discussions. Merging into https://github.com/tailwindlabs/tailwindcss/discussions/6031

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

We'll add it for the next release: https://github.com/tailwindlabs/tailwindcss/pull/10294
